<launch>
	<arg name="record_bag" default="false" />
    <rosparam file="$(find px4_base)/config/config.yaml" command="load" />

    <node name="supervisor"  pkg="px4_base" type="supervisor.py" output="screen" >
        <param name="offset_x" value="5.0"/>
        <param name="offset_y" value="8.0"/>
        <param name="offset_z" value="0.0"/>
        <param name="rotation" value="0.0" />
    </node>
	
	<node pkg="rosbag" type="record" name="rosbag_record"
        if="$(arg record_bag)"
        args="record -a -o $(find px4_base)/bags/" />

</launch>
